GOVERNANCE.md

```diff
@@ -0,0 +1,55 @@
+The **WGPU project** is a set of open-source libraries that _enables application
+authors to write portable and performant graphics programs_. It was originally
+conceived to provide an implementation of WebGPU for Firefox as the standard
+evolved and settled into something that could be shipped on all web browsers.
```

Comment:
```suggestion
authors to write portable and performant graphics programs_. It was originally
conceived to provide an implementation of WebGPU for Firefox, and 
settled into something that could be shipped on all web browsers.
```

The original feels a bit run-on-ey

Reply:
Implemented as suggested with e71c68132.